### DISCUSSION TITLE
Disable combobox zoom when focused in PWA

### DISCUSSION THREAD

**Discussion by dorskie** [unovue/reka-ui#731] [Q&A]
Good day, I am using radix-vue combobox on my PWA and it automatically zooms in when focused. already implemented <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" /> and it still the same. 
Is there a way to disable it?

**Comment by wanxe** [comment]
It happens exactly the same
